ios/src/TGGeoPoint.h

```diff
@@ -0,0 +1,27 @@
+//
+//  TGGeoPoint.h
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#ifndef TGGeoPoint_h
+#define TGGeoPoint_h
```

Comment:
Is this the only header here that uses include guards? If we need these then we should prefer `#pragma once`, but if we can use `#import` for headers on iOS then it seems that these shouldn't be needed.